src/docs/performance-monitoring/performance/event-detail.mdx

```diff
@@ -82,7 +82,7 @@ import sentry_sdk
 
 #  ...
 
-columns = ["first_column", "second_column", "third_columns", "fourth_column", "fifth_column", "sixth_colummn"]
+columns = ["first_column", "second_column", "third_columns", "fourth_column", "fifth_column", "sixth_column"]
```

Comment:
@joshuarli for a future pass:

```diff
-third_columns
+third_column
```

:)

I've done a similar mass spellchecking in Django docs in a hackday some years ago, good stuff!